packages/plugin-infra/src/types.ts

```diff
@@ -181,6 +181,8 @@ export namespace Plugin.Shared {
         openPopupWindow(route?: PopupRoutes, params?: Record<string, any>): Promise<void>
         /** Close popup window */
         closePopupWindow(): Promise<void>
+        /** Open popup connect window */
+        openPopupConnectWindow(): Promise<void>
```

Comment:
Need `closePopupConnectWindow()` as well.

Reply:
No, the `closePopwindow` can meet your need.